src/mpd.c

```diff
@@ -3193,6 +3212,291 @@ mpd_command_update(struct evbuffer *evbuf, int argc, char **argv, char **errmsg,
   return 0;
 }
 
+static void
+mpd_add_idle_events(short event_mask);
+
+enum mpd_sticker_cmd_id {
+  MPD_STICKER_GET,
+  MPD_STICKER_SET,
+  MPD_STICKER_DELETE,
+  MPD_STICKER_LIST,
+  MPD_STICKER_FIND,
+  MPD_STICKER_INVALID
+};
+
+struct mpd_sticker_cmd {
+  const char *cmd;
+  enum mpd_sticker_cmd_id id;
+  int need_args;
+  int want_dir;
+};
+
+static struct mpd_sticker_cmd mpd_sticker_cmd[] = {
+  { "get", MPD_STICKER_GET, 5, 0 },
```

Comment:
Instead of defining an enum for the different sticker actions, define a "handler" function (same concept that is used for mpd commands and sendmessage/channel).